setup.py

```diff
@@ -18,7 +18,7 @@
     "dateparser==1.2.0",
     "duckdb==1.0.0",
     "enforce_typing",
-    "eth-account==0.11.0",
```

Comment:
please remove the eth-account dependency altogether, it is implicit from web3.py. I can do that if needed.